src/app.js

```diff
@@ -0,0 +1,92 @@
+import axios from "axios";
+
+console.log("It's nice to see you smile!");
+
+//make reference to <div> tag in html and add <h1> to it with a Country Name!
+const countryResults = document.getElementById("country-results");
+//make reference to <div> tag in html and add <img> to it with an image
+const worldImage = document.getElementById("world-image");
+//make reference to <form> tag in html
+const searchForm = document.getElementById("search-form");
+
+//put event listener on searchForm
+searchForm.addEventListener( "submit", fetchOneCountry);
+
+
+
+//1. write async function that gets information about ALL countries with a GET request, and log the results in the console
+//now get information for only one country, for example Norway
+
+async function fetchAllCountries(name) {
+    try {
+        const result = await axios.get(`https://restcountries.com/v2/name/${name}`);
+        const countryDetails = result.data[0];
```

Comment:
Je zou hier ook het volgende van kunnen maken
```suggestion
        const { flag, name, population, region, capital, currencies, languages } = result.data[0];
```
Dan hoef je vervolgens nergers meer `countryDetails.` vóór te zetten